KsApi/mutations/UpdateBackingMutation.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+public struct UpdateBackingMutation<T: GraphMutationInput>: GraphMutation {
+  var input: T
+
+  public init(input: T) {
+    self.input = input
+  }
+
+  public var description: String {
+    return """
+    mutation updateBacking($input: UpdateBackingInput!) {
+      updateBacking(input: $input) {
+        checkout {
+          state
```

Comment:
Hey Justin! I just tested @cj-dimaggio's changes in `createBacking` and I know this is `updateBacking` but since the mutations will now be synchronous, we should be checking that the state is `successful`

Reply:
Cool thanks!

Reply:
@eoji @cj-dimaggio do you mean that if there's an error the checkout state will be `.failed`? Would it make more sense to return an error via the `errors` array in that case? 🤔